content/gsoc/gsoc-2024/_index.md

```diff
@@ -0,0 +1,64 @@
+4. We have [active](//www.honeynet.org/node/371) volunteer member [chapters](//www.honeynet.org/og) in many countries and from many different backgrounds, with a wide variety of skills and experience they are happy to share
+5. We have always been committed to the concepts of open source software and freely share everything we do, including each chapter publishing regular public [status reports](//www.honeynet.org/chapter/statusreports) on their recent activity
+6. We maintain active public and private communities of developers and researchers who use and contribute to our tools each day
+7. We provide our members and the community with the public and private infrastructure necessary to support distributed collaborative remote working, such as IRC channels, mailing lists, subversion repositories, Trac instances for ticket management and wikis, content management systems, blogs, live deployments with real end users for testing and regular feedback, etc.
+8. We are hands on, supportive and keen to involve more talented people in projects we are really passionate about
+9. We have a strong track history of mentoring new members and successfully delivering open source projects, tools and research that demonstrably benefit the community
+10. The vast majority of our GSoC projects were delivered successfully and our students were happy, with a number of the tools created going on to become widely used within the security community
+11. Students from GSoC have gone on to become active members of the honeynet community, including proposing project ideas for future GSoCs or offering to be project mentors and organization administrators (so we can't be too unpleasant a bunch to get involved with!) ;-)
+12. We have ex-GSoC students actively serving on our Board of Directors and in Executive Officer positions, so GSoC has definitely proven to be a great career path for students wanting to get more involved in open source and information security R&D.
+13. Honeypots and honeynet technology, research and tools have filtered down benefits to many areas of IT, web development, operational service management, Internet education and computer security research
+
+### Other ways to get involved
+
+[GSoC](//summerofcode.withgoogle.com/) is not the only way you can become involved in the with honeynet technologies and open source software - check out our [current](//www.honeynet.org/challenges) or [historic](//old.honeynet.org/misc/chall.html)series of [forensic challenges](//www.honeynet.org/challenges). Or learn more about the practical, real world application of honeynet technology in our popular series of ["Know Your Enemy"](//www.honeynet.org/papers) whitepapers, which now include projects and tools output from previous GSoCs students or mentors such as [PicViz](//www.honeynet.org/node/499), [Glastopf](//www.honeynet.org/papers/KYT_glastopf), [Qebek](//www.honeynet.org/papers/KYT_qebek) or [Conficker](//www.honeynet.org/papers/conficker).
```

Comment:
This also needs to be cleaned up. I can also help with non-obvious issues after we merged.